wikipedia-sentences/flows/query.yml

```diff
@@ -0,0 +1,18 @@
+!Flow
+with:
+  read_only: true  # better add this in the query time
+  rest_api: true
+  port_expose: $JINA_PORT
+pods:
+  encoder:
+    uses: pods/encode.yml
+    parallel: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  doc_indexer:
+    uses: pods/doc.yml
+    shards: $JINA_SHARDS
+    separated_workspace: true
+    polling: all
+    uses_reducing: _merge_all
```

Comment:
_merge_all is deprecated, just in advance

Reply:
Fixed